config/locales/idv/fr.yml

```diff
@@ -191,7 +200,7 @@ fr:
     titles:
       activated: Votre identité a déjà été vérifiée
       cancel: Nous ne pouvons pas vérifier votre identité
-      come_back_later: NOT TRANSLATED YET
+      come_back_later: Revenez vite.
```

Comment:
The English text here is "come back soon". I see that it's related to resending a USPS letter. Can you remind me what the context is where we want the user to come back soon? The French translation makes it sound like "come back quickly", which is probably not what we want.

Reply:
This shows up after they go through IdV with USPS. The text is telling them to come back once they have their code.